docs/Guide/application-commands/application-command-registry/registering-context-commands.mdx

```diff
@@ -97,68 +87,25 @@ export class SlashCommand extends Command {
-    // Use `isUserContextMenu` to ensure this command was executed as a User Context Menu Command.
-    if (interaction.isUserContextMenu()) {
-    }
+    // ...
   }
 }
 ```
 
-## Context Menu Command Registry Methods
-
-:::warning
+:::caution
 
-To register a context menu input command, you need to acquire the [`ApplicationCommandRegistry`][acr]. Please see the
-[`ApplicationCommandRegistry`][aaacr] section for more details.
+Just like the Message Context Menu Command, the [`registerContextMenuCommand`][rcmc] method takes in a second options
+parameter. Please see [Chat Input Command Registry Options][cicro], since there are important details not repeated here.
 
 :::
 
-### `registerContextMenuCommand`
-
-The method which is responsible for telling the registry that this command should be registered. It takes two
-parameters:
-
-- `command`: The Context Menu Command data to be registered. This can be a `discord.js`
-  [`ApplicationCommandData`][djs-command-data-docs] object a `@discordjs/builders`
-  [`ContextMenuCommandBuilder`][djs-context-menu-command-builder-docs] or a method which returns the
-  `@discordjs/builders `[`ContextMenuCommandBuilder`][djs-context-menu-command-builder-docs].
-
-  For more details on how to use
-  `@discordjs/builders `[`ContextMenuCommandBuilder`][djs-context-menu-command-builder-docs], please see
-  [`discord.js guide`][djs-context-menu-command-builder-docs].
-
-- `options` These are optional registry [`options`][cicro] you can provide when registering the command you provided. By
-  default, this is set to `{ behaviorWhenNotIdentical: RegisterBehavior.Overwrite }`.
+## Implementing a Context Menu Command
 
-An example of this method was shown above at [How to register a Message Context Menu Command][inner-ref-how-to].
+Now that we've covered registering a Chat Input Command with Discord, the last step is to implement the `contextMenuRun`
+method for the [`Command`][command] class so your bot can respond! The example here is the second User Context Menu
+Command started above:
 
-### `addContextMenuCommandNames`
-
-The command names which let the registry know that these names should trigger the registry. These names doesn't register
-the command but only let the registry know that these names should trigger the command, i.e. the names are only stored
-in command store to be used by the registry. Names can be passed as strings.
-
-### `addContextMenuCommandIds`
-
-The command ids that informs the registry to trigger it for the specified ids only. Ids can be passed as strings.
-
-:::warning
-
-`addContextMenuCommandNames` or `addContextMenuCommandIds` should only be used when you don't want `Sapphire` to manage
-the application commands automatically for you else consider using `idHints` instead.
-
-Whenever the `addContextMenuCommandIds` method is called with an invalid id (we expect a string that can be turned into
-a BigInt), then it will **ALWAYS** warn in the console. No, this cannot be turned off.
-
-:::
```

Comment:
These are not referenced on any guide page anymore after this change.

Reply:
Moved to "Routing existing Application Commands to a Command"